src/infra/alarm/template.yaml

```diff
@@ -750,3 +758,177 @@ Resources:
       Dimensions:
         - Name: service
           Value: StatusRetrieverFunction
+
+#
+# CloudWatch alarms for API Gateway
+#
+
+  ApiLatencyP95Taking2SecsOrLonger:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Over 5% of API Gateway requests are exceeding 2 seconds in a 10 minute period. ${RunBookURL}"
+      Namespace: AWS/ApiGateway
+      MetricName: Latency
+      ComparisonOperator: GreaterThanThreshold
+      ExtendedStatistic: p95
+      Threshold: 2000
+      EvaluationPeriods: 1
+      Period: 600
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: ApiName
+          Value: !Ref PrivateRestApiName
+
+  ApiLatencyP99Taking2SecsOrLonger:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: "Over 80% of API Gateway requests are exceeding 2 seconds in a 10 minute period."
+      Namespace: AWS/ApiGateway
+      MetricName: Latency
+      ComparisonOperator: GreaterThanThreshold
+      ExtendedStatistic: p80
+      Threshold: 2000
+      EvaluationPeriods: 1
+      Period: 600
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: ApiName
+          Value: !Ref PrivateRestApiName
+
+  ApiHttp5xxErrors:
```

Comment:
@domantas-gds - did you manage to trigger this in the end?

Reply:
Yes! Thanks for your help